src/styles/screen.scss

```diff
@@ -1,16 +1,53 @@
-@import './variables.scss';
+@import "tailwindcss/base";
+
+html {
+  @apply font-body;
+}
+
+
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
 
 .button-knockout {
-  border: 1px solid $fill_color;
-  padding: 5px 10px;
-  background: #FFF;
-  transition: color 300ms $cubic, background-color 300ms $cubic;
-  height: 100%;
-  font-size: $font_size;
+  @apply border px-4 h-full bg-white border-gray-900 transition-colors
+  duration-300;
 
   &.selected {
-    background: $fill_color;
-    border: 1px solid $fill_color;
+    @apply bg-gray-900;
     color: #FFF;
   }
+}
+
+.fade-in-down-enter-active,
+.fade-in-down-leave-active {
+  @apply transition-all duration-300 ease-in-out;
+}
+
+.fade-in-down-enter,
+.fade-in-down-leave-to {
+  @apply opacity-0 -translate-y-3 transform;
```

Comment:
So succinct 🤓

Reply:
It should be doing a lil bit more finesse here but I can't figure out why it's failing to transition transforms :D I think it's a vue transition issue.